torch/csrc/distributed/c10d/TraceUtils.h

```diff
@@ -394,6 +400,8 @@ struct NCCLTraceBuffer {
     // group)
     size_t seq_id_;
     size_t op_id_;
+    uint64_t nccl_comm_hash_; // commHash used inside NCCL to correlate with
```

Comment:
Note flight recorder needs to work when we not using NCCL_EXP, in which case uid_ is more generic than comm_hash?

Reply:
Thanks for pointing this out. I think when we are not using NCCL_EXP, no commHash and nccl dump will be available, thus we can remove them. I will make them only available when NCCL_EXP and NCCL_COMM_DUMP are defined.